functions/graphql/schema/flightRecord.js

```diff
@@ -31,28 +31,18 @@ module.exports.schema = gql`
   input FlightStationInput {
     name: String! # The name of the station. Required
     badges: [ID]! # The list of badges that the station earned. Includes missions
-    userId: ID # ID of the user who was at this station for this flight
```

Comment:
Why was this removed? It is a crucial bit of data that isn’t extended anywhere else. I don’t even know if you can extend inputs.